### ISSUE TITLE
Instance seems to start but is inaccessible

### ISSUE THREAD

**Issue by bobbyl140** [DuoStream/Duo#141]
I made a new instance using the GitHub version, left all the options as default, and pressed run, and while everything seems to be running, the "Open Web User Interface" button for the instance is grayed out, and can't be visited from the Global web UI or by manually typing the URL. I can't connect to it via Moonlight either. No logs show up, and in fact the `config` folder is never created. I tried running `DuoRdp.exe` as specified [here](https://github.com/DuoStream/Duo/issues/128#issuecomment-2197884711), and that results in a `getaddrinfo failed` error. I searched existing issues and didn't see anything specifically about that. Any help is appreciated! Thank you.

**Comment by mdusch** [comment]
Having the same issue, server is running, the service is running, I can connect through both web clients but cannot connect to the instance through a streaming client (moonlight).

**Comment by bobbyl140** [comment]
> I can connect through both web clients

You mean the sunshine UI is reachable for you? For me not even that works. Did any of the other issues seem like what you’re experiencing?

**Comment by mdusch** [comment]
> > I can connect through both web clients
> 
> You mean the sunshine UI is reachable for you? For me not even that works. Did any of the other issues seem like what you’re experiencing?

I can reach sunshine web UI and even add a device with a pin but that process either times out or adds successfully then the server is unreachable by that client. Tried uninstalling and creating new servers with no luck.

**Comment by bobbyl140** [comment]
Huh, I’m not sure. That may or may not be related, I guess let’s wait for the dev to reply

**Comment by mdusch** [comment]
Hm YMMV but I enabled gamestream and sunshine started working (didn't even error out in the web UI). I disabled it after it started working and it still worked. Strange behavior.

**Comment by Chanch0** [comment]
"Open Web User Interface" grayed out for me as well. Worked fine before.

**Comment by bobbyl140** [comment]
After letting it sit for *a very long time* the web UI eventually began working.

**Comment by Black-Seraph** [comment]
You probably have a Microsoft account associated with your local account.

This forces the account to authenticate via a different mechanism which is not compatible with Duo.

tl;dr: Don't link Microsoft accounts to local accounts, it will break them.

**Comment by bobbyl140** [comment]
Before installing I removed all traces of my Microsoft account though. Are they *permanenlty* broken?
